spec/lib/jobs/create-obm-settings-spec.js

```diff
@@ -65,12 +69,45 @@ describe(require('path').basename(__filename), function () {
 
         it('should run <obmService>.powerStatus in liveTestObmConfig()', function() {
             var self = this;
+
             var powerStatus = self.sandbox.stub(VboxObmService.prototype, 'powerStatus');
             powerStatus.resolves();
             return self.job.liveTestObmConfig()
             .then(function() {
                 expect(powerStatus).to.have.been.calledOnce;
             });
         });
+
+        it('should decrypt passwords when testing OBM config', function() {
+            var self = this;
+
+            var ObmService = helper.injector.get('Task.Services.OBM');
+            self.sandbox.stub(ObmService, 'create').returns({
+                powerStatus: Promise.method(function() {})
+            });
+
+            var jobOptions = {
+                service: 'ipmi-obm-service',
+                config: {
+                    alias: 'test',
```

Comment:
Why add `alias` here? I think this may confuse user if user reads the unit-test to study how to use this function.